internal/lm/resource.go

```diff
@@ -298,7 +298,10 @@ func getArchFamily(computeMajor, computeMinor int) string {
 		}
 		return "turing"
 	case 8:
-		return "ampere"
+		if computeMinor < 9 {
+			return "ampere"
+		}
+		return "ada"
```

Comment:
Should this be `ada-lovelace`? I saw your comment elsewhere on the CUDA samples using just `Ada` but I'm not sure if this is the correct source of truth to go by.

Reply:
We also have: https://github.com/NVIDIA/go-nvlib/blob/bf0cbc7fef6c51b968da7abd1e8788f0647b2266/pkg/nvlib/device/device.go#L89

Reply:
OK. I don't feel too strongly - it mostly just feels inconsistent. Especially since its shortened to "Ada" and not "Lovelace" since all others are the last name, not the first name.

Reply:
Updated to `ada-lovelace` and created https://github.com/NVIDIA/go-nvlib/pull/49